.env

```diff
@@ -0,0 +1 @@
+PYTHONPATH=src
```

Comment:
`.env` usually holds local secrets (like service tokens) and therefore should not be uploaded to git.